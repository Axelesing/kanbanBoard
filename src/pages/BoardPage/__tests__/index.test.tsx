import { render } from '@testing-library/react'

import { BoardPage } from '..'

jest.mock('@/features/kanban', () => ({
  Board: () => <div>Board</div>,
}))

jest.mock('@/widgets/Modal', () => ({
  TaskModal: () => <div>TaskModal</div>,
}))

describe('BoardPage', () => {
  test('matches snapshot', () => {
    const { baseElement } = render(<BoardPage />)

    expect(baseElement).toMatchSnapshot()
  })
})
