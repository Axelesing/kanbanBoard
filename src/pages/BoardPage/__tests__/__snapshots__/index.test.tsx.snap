// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`BoardPage matches snapshot 1`] = `
<body>
  <div>
    <div>
      Board
    </div>
    <div>
      TaskModal
    </div>
  </div>
</body>
`;
